airflow/config_templates/default_celery.py

```diff
@@ -36,6 +36,14 @@ def _broker_supports_visibility_timeout(url):
     if _broker_supports_visibility_timeout(broker_url):
         broker_transport_options['visibility_timeout'] = 21600
 
+result_backend = str(conf.get('celery', 'RESULT_BACKEND'))
+if result_backend.startswith(('postgresql://', 'sqlite:///', 'mysql://')):
```

Comment:
we also support mssql: `mssql+pyodbc://`
and legacy postgres scheme (for older version of sqlalchemy) is i think `postgres://`